### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by osbentech**
this is a React project named math-magician. It enables user run arithmetic operation.
below are feature I implemented : 
- Enable event handlers

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (andersonlebon) [inline src/components/Calculator.js:4]: - [ ] kindly make sure you solve linters instead of disabling them

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+function Calculator() {
```
Comment (andersonlebon) [inline src/components/Calculator.js:4]: ## [OPTIONAL]
- [ ] Kindly i think that using arrow function would sound good here as we are trying to use ES6

File: src/components/Calculator.js (hunk 1 above)
Comment (andersonlebon) [inline src/components/Calculator.js]: - [ ] kindly make sure you solve linters instead of disabling them

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @osbentech

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @osbentech, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 
You might also consider implementing the previous reviewer's optional suggestions as they would help you write better code.



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**